src/packages/verticaltabs/doc.en-US.md

```
@@ -0,0 +1,135 @@
+
+| Property | Description | type | Default |
+| --- | --- | --- | --- |
+| title | title | `string` | `-` |
+| value | tag Key , matching identifier, default is index value | `string` \| `number` | `-` |
+| disabled | Whether to disable the label | `boolean` | `false` |
+
+## Theming
+
+### CSS Variables
+
+The component provides the following CSS variables, which can be used to customize styles. Please refer to [ConfigProvider component](#/en-US/component/configprovider).
```

Markdownlint

Link fragments should be valid

(MD051, link-fragments)